cypress/e2e/tests/custom-metrics/assessment_metrics.test.ts

```diff
@@ -0,0 +1,79 @@
+import {
+    createMultipleApplications,
+    deleteApplicationTableRows,
+    login,
+} from "../../../utils/utils";
+import { Assessment } from "../../models/migration/applicationinventory/assessment";
+import { Metrics } from "../../models/migration/custom-metrics/custom-metrics";
+import { Stakeholders } from "../../models/migration/controls/stakeholders";
+import * as data from "../../../utils/data_utils";
+
+let applicationList: Array<Assessment> = [];
+let metrics = new Metrics();
+const stakeholdersList: Array<Stakeholders> = [];
+const stakeholdersNameList: Array<string> = [];
+
+describe(["@tier2"], "Custom Metrics - The total number of initiated assessments", function () {
+    before("Login and create test data", function () {
+        // Perform login
+        login();
+
+        // Navigate to Application inventory tab, delete all applications
+        deleteApplicationTableRows();
+
+        // Navigate to stakeholders control tab and create new stakeholder
+        const stakeholder = new Stakeholders(data.getEmail(), data.getFullName());
+        stakeholder.create();
+        cy.wait(2000);
+
+        stakeholdersList.push(stakeholder);
+        stakeholdersNameList.push(stakeholder.name);
+
+        // Create 2 applications
+        applicationList = createMultipleApplications(2);
+    });
+
+    it("Perform Assessment-Validate metrics assessment count increased", function () {
+        let count = applicationList.length;
+
+        // Perform assessment of application
+        applicationList[0].perform_assessment("low", stakeholdersNameList);
+        cy.wait(2000);
+        applicationList[0].verifyStatus("assessment", "Completed");
+
+        // Reload the metrics page
+        metrics.reloadPage();
+
+        // Validate the application inventory count increased to 1
+        metrics.validateAssessmentInitiated(count--);
```

Comment:
The count should increase here , not decrease .

Reply:
Actually here count=ApplicationList.length. I just performed assessment on 1 app. 
Will perform assessment on both apps.